core/src/main/java/org/infinispan/remoting/rpc/RpcManager.java

```diff
@@ -42,9 +119,14 @@
     *                   not change it. Any change in {@link RpcOptions} during a remote invocation can lead to
     *                   unpredictable behavior.
     * @return  a map of responses from each member contacted.
+    * @deprecated Since 9.2, please use {@link #invokeCommand(Collection, ReplicableCommand, ResponseCollector, RpcOptions)} instead.
     */
    Map<Address, Response> invokeRemotely(Collection<Address> recipients, ReplicableCommand rpc, RpcOptions options);
```

Comment:
you forgot to annotate the method as `@deprecated`. 
There are still usages in `LocalStreamManagerImpl` and `BiasManagerImpl`. Do you want to "fix" it in this PR or later?

Reply:
edit: and `XSiteAdminOperations`

Reply:
Initially I intended to replace both the usages of `invokeRemotely()` and `invokeRemotelyAsync`. But now I see there are quite a few usages of `invokeRemotely()` remaining (or maybe added since I first created the PR), so I'm leaving them for a separate PR. 

I did replace the usages of `invokeRemotelyAsync()` in `LocalStreamManagerImpl`, `BiasManagerImpl`, and `TransactionDecodeContext`.